src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -298,7 +299,8 @@ func (cfg *Config) applyClientDefaults() {
 
 	if cfg.AutoLaunch == nil {
 		cfg.AutoLaunch = new(bool)
-		*cfg.AutoLaunch = false
+		*cfg.AutoLaunch = true
+		launcher.CreateLaunchFile(*cfg.AutoLaunch)
```

Comment:
We want two things here. First, I think it makes sense to make auto-launch the default, especially because users have had trouble finding Lantern once it's installed. Second, we want to reflect whatever the starting state is on startup, as otherwise we won't set the registry setting or plist until the user actually interacts with the UI. Come to think of it, though, that call should be outside of the check for nil -- fixing now.